sites.json

```diff
@@ -571,6 +571,16 @@
             "blueapron.com"
         ]
     },
+    
```

Comment:
Besides this whitespace, this looks good to merge. If you could remove it, that would be great. Otherwise, I'll push an update to your branch to fix it and merge later today.

Thanks in advance!